engine/runtime-integration-tests/src/test/java/org/enso/logging/service/logback/telemetry/test/TestLogFormatter.java

```diff
@@ -0,0 +1,124 @@
+package org.enso.logging.service.logback.telemetry.test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import org.enso.logging.service.logback.telemetry.LogFormatter;
+import org.enso.logging.service.logback.telemetry.LogMessage;
+import org.junit.Test;
+
+public class TestLogFormatter {
+  @Test
+  public void shouldNotTransformLog_WithIncorrectMessage() {
+    var logMessage = createLogMessage("Message - arg=1", 1);
+    var json = LogFormatter.transform(logMessage);
+    assertThat("was not transformed", json, is(nullValue()));
+  }
+
+  @Test
+  public void shouldNotTransformLog_WithoutArguments() {
+    var logMessage = createLogMessage("Message: arg=1");
+    var json = LogFormatter.transform(logMessage);
+    assertThat("was not transformed", json, is(nullValue()));
```

Comment:
I suggest to introduce new:
```
assertMessage("Message", "arg", 1)
```
and use it to check:
- name of the message
- argument names
- values of each argument

Reply:
Fixed in 9c7f2b4cd0298afa57cf649f0dc77e248740c13b